models/spike_generator.h

```diff
@@ -249,18 +250,22 @@ class spike_generator : public DeviceNode
     bool precise_times_;
 
     //! Allow and round up spikes not on steps; irrelevant if precise_times_
+    bool allow_offgrid_times_;
+    //! Same as allow_offgrid_times_, here for backward compatibility
```

Comment:
Keep only `allow_offgrid_times_` internally. We need backward compatibility of the *user interface*, not of the implementation.